### PR TITLE
Refactoring the Airnote dashboard to use data from Notehub directly

### PR DESCRIPTION
**Pull request by tjvantoll**
This PR changes the airnote.live site to use data from the Notehub API instead of a Safecast-based endpoint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,37 @@ open-source reference implementation of a web-based
 
 ## Local Development
 
-First install the [Node.js runtime](https://nodejs.org/en/).
+The airnote.live site runs on top of Node.js, so as such you’ll need to start by [installing Node.js](https://nodejs.org/en/download/) if you haven’t already.
+
```
Comment (paigen11) [inline README.md:10]:
```suggestion
The airnote.live site runs on top of Node.js, so as such you’ll need to start by [installing Node.js](https://nodejs.org/en/download/) if you haven’t already.

It also uses npm as its package manager, so [add npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) as well if it's not already installed locally.

```

File: README.md [hunk 2]
```diff
@@ -6,14 +6,37 @@ open-source reference implementation of a web-based
...
 ```sh
-npm install # get dependencies
-npm run dev # start a dev server
+npm run dev
+```
+
+## Web App
+
+You can access the airnote.live site locally at `http://localhost:5555/`. To view a specific device’s dashboard, you’ll need to use a URL of `http://localhost:5555/<device_uid>/dashboard`.
+
```
Comment (paigen11) [inline README.md:26]: Should probably also add an addition about the need for `device: { deviceUID: "dev:864475044215258" pin: "1234" productUID: "product:org.airnote.solar.v1" }` in the local storage as well.

The missing PIN kept my dashboard from working.

File: src/services/device.js [hunk 3]
```diff
@@ -46,41 +47,53 @@ function getHistory(readings) {
...
-
-  return fetch('https://airnote-api.onrender.com/?' + new URLSearchParams({
-    device_uid: deviceUID,
-    from: eightDaysAgo.toISOString(),
-    to: new Date().toISOString()
-  }))
-    .then(response => response.json())
-    .then(data => {
+  return fetch(
+    "http://localhost:3001/?" +
```
Comment (paigen11) [inline src/services/device.js:51]: Will this hardcoded URL work in prod or does it need to be more dynamic?

**Review by paigen11 — APPROVED** [review]
Looks good - just a few minor comments and questions I had

File: src/services/device.js (hunk 3 above)
Comment (tjvantoll) [reply src/services/device.js:51]: Ah yeah I totally forgot about this. It definitely needs to be dynamic. I’ll take a stab at this today.

Comment (tjvantoll) [reply src/services/device.js]: Ok here’s my fix for that ad03d27390ce2fe66624f1956c85ea6aca5c3f3f.

The Prettier auto formatting makes this messy but the key part is this bit in `rollup.config.js`. This substitution was already happening for the Notehub API which made adding this one pretty easy.

```
      "process.env.SERVER_URL": JSON.stringify(
        production
          ? "https://airnote-api.onrender.com"
          : "http://localhost:3001"
      ),
```

**Comment by tjvantoll** [comment]
This was included in 1c80d54c86b53814bd650943d1e0c29bd3b8045c.

**Pull request closed without merging**